checker/src/org/checkerframework/checker/index/upperbound/UpperBoundUtil.java

```diff
@@ -0,0 +1,166 @@
+package org.checkerframework.checker.index.upperbound;
+
+/*>>>
```

Comment:
The Checker Framework is now compiled with Java 8, so annotations don't have to be in comments.  (Unless they are annotations in the framework package that are declared in the checker package.)  [The manual](https://checkerframework.org/manual/#uncommenting-annotations) gives Unix commands to easily uncomment the annotations.